Azure/sonic-utilities
# [vlan] remove dhcp-relay as dhcp-relay commands will come as a plugin

**- What I did**
Remove dhcp relay commands from sonic-utilities. dhcp-relay commands will come as a plugin with dhcp-relay docker installation.
See https://github.com/Azure/SONiC/pull/682

**- How I did it**
Remove dhcp-relay commands from vlan. Make "show vlan brief" command table output extendable.

**- How to verify it**
Install dhcp-relay docker as app.ext. Verify that "config vlan dhcp-relay" and "show vlan brief" show dhcp data.

**- Previous command output (if the output of a command-line utility has changed)**

**- New command output (if the output of a command-line utility has changed)**
